.rubocop.yml

```diff
@@ -40,6 +40,10 @@ Style/BlockDelimiters:
   Description: Prefer braces for chaining. Mostly an aesthetical choice. Better to
     be consistent then.
   EnforcedStyle: braces_for_chaining
+Style/BracesAroundHashParameters:
+  Description: Braces are required by Ruby 2.7. Cop removed from RuboCop v0.80.0.
+    See https://github.com/rubocop-hq/rubocop/pull/7643
+  Enabled: true
```

Comment:
Is this right?? Seems to directly contradict the change made in PDK.
https://github.com/puppetlabs/pdk-templates/pull/329

Reply:
Ignore this comment.  It doesn't relate to this PR. Rebasing/merging must have gone a bit funny at some point to pull this in here.